tests/foreman/cli/test_registration.py

```diff
@@ -295,3 +295,37 @@ def test_positive_custom_facts_for_host_registration(
     for interface in interfaces:
         for interface_name in interface.values():
             assert interface_name in str(host_info['network-interfaces'])
+
+
+@pytest.mark.rhel_ver_list([settings.content_host.default_rhel_version])
+def test_positive_global_registration_with_gpg_repo(
```

Comment:
this looks a good candidate for e2e and upgrades?

Reply:
Could be good for upgrades but not for e2e imo.